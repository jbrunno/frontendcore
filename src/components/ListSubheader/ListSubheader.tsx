import React, { ElementType } from 'react';
import {
  ListSubheader as MuiListSubheader,
  ListSubheaderProps as MuiListSubheaderProps,
} from '@mui/material';

export type ListSubheaderProps = MuiListSubheaderProps & {
  component?: ElementType;
};

export function ListSubheader(props: ListSubheaderProps) {
  return <MuiListSubheader {...props} />;
}
export default ListSubheader;
