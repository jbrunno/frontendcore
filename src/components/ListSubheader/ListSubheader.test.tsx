import React from 'react';
import { render } from '@test-utils/component-testing';
import ListSubheader from './ListSubheader';

test('renders the ListSubheader', () => {
  const { container } = render(<ListSubheader />);
  expect(container).toMatchSnapshot();
});
