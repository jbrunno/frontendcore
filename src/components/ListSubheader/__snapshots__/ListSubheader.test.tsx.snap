// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ListSubheader 1`] = `
.emotion-0 {
  box-sizing: border-box;
  line-height: 48px;
  list-style: none;
  color: #3B3B4199;
  font-family: Lato,sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  padding-left: 16px;
  padding-right: 16px;
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  z-index: 1;
  background-color: #fff;
}

<div>
  <li
    class="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky emotion-0"
  />
</div>
`;
